src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -7,12 +7,21 @@
 
 public class Navigate extends BasePageObject {
 
+  /**
+   * Adding default testing group is invoked explicitly in URL as temporary change until
+   * the end of experiment (01.07.2016) described in DAT-4483.
+   */
+
+  private static final String defaultTestingGroup =
+      "&ABTEST.MERCURY_VIEWABILITY_EXPERIMENT=DEFAULT";
+
   public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") ||
                          pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+    String abGroup = pageName.equals("") || pageName.equals("/") ? "" : defaultTestingGroup;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "/" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)